README.md

```diff
@@ -207,7 +207,7 @@ export default class CounterApp {
 
 #### Decorators
 
-The `@connect` decorator lets you create smart components less verbosely:
+The `@connect` decorator lets you create smart components more succinctly:
```

Comment:
i actually think "succinctly" is the one of the most succinct ways to write this. "concisely" could also work.